master/internal/db/postgres_tasks_intg_test.go

```diff
@@ -10,12 +10,13 @@ import (
 	"time"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/determined-ai/determined/master/pkg/etc"
-	"github.com/determined-ai/determined/master/pkg/model"
-	"github.com/determined-ai/determined/master/pkg/ptrs"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/guregu/null.v3"
+
```

Comment:
take out? I think @stoksc  is working on fixing this

Reply:
although if this is not tripping up the CI could we just commit this to master? cc @stoksc

Reply:
I was about to commit it also, idc who does it

Reply:
I might... this was what happened when I did `make fmt` though.